apps/andi/lib/andi/dataset_util.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Andi.DatasetUtil do
+  @moduledoc """
+  An Abstraction that handle the specifics of the Brook View state for andi datasets.
+  """
+
+  import Andi
+
+  @collection :dataset
+
+  def delete(id) do
+    Brook.ViewState.delete(@collection, id)
+  end
+end
```

Comment:
Can we put this under Andi.Services?  Maybe consolidate this with the DatasetRetrieval?

Reply:
After discussion - we'll address this in a separate card.